Sources/TuistCore/Utils/OtoolController.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import RxSwift
+import TSCBasic
+import TuistSupport
+
+enum OtoolErrors: FatalError, Equatable {
+    case invalidOutput
+
+    var description: String {
+        switch self {
+        case .invalidOutput: return "The output is not in the format tuist expects"
```

Comment:
If we put ourselves in the shoes of the users and we see this error, would we know what to do? It's important that users contain enough information for the user to know what happened and what they can do to overcome it. If we don't hint anything of that, the users will most likely show up in our Slack channel or open an issue saying: I got this and I don't know what to do. That'll increase our support load and that's something we can avoid with a more descriptive message. 

I propose the following:

```
We couldn't get the dependencies of binary /path/to/binary using otool because the output has an unexpected format. 
```

Note that if a user brings that up, we'll immediately know what's going on and we won't have to spend much time debugging the issue.

I'd also recommend adding a `debug` log with the raw output from `otool`.